doc/source/whats_new.rst

```diff
@@ -918,19 +920,19 @@ Version 0.2
 Changelog
 ~~~~~~~~~
 
-   - New stats functions for FDR correction and Bonferroni by `Alex Gramfort`_.
+    - New stats functions for FDR correction and Bonferroni by `Alex Gramfort`_.
 
-   - Faster time-frequency using downsampling trick by `Alex Gramfort`_.
+    - Faster time-frequency using downsampling trick by `Alex Gramfort`_.
 
-   - Support for volume source spaces by `Alex Gramfort`_ (requires next MNE release or nightly).
+    - Support for volume source spaces by `Alex Gramfort`_ (requires next MNE release or nightly).
 
-   - Improved Epochs handling by `Martin Luessi`_ (slicing, drop_bad_epochs).
+    - Improved Epochs handling by `Martin Luessi`_ (slicing, drop_bad_epochs).
 
-   - Bug fix in Epochs + ECG detection by Manfred Kitzbichler.
+    - Bug fix in Epochs + ECG detection by Manfred Kitzbichler.
 
-   - New pick_types_evoked function by `Alex Gramfort`_.
+    - New pick_types_evoked function by `Alex Gramfort`_.
 
-   - SourceEstimate now supports algebra by `Alex Gramfort`_.
+    - SourceEstimate now supports algebra by `Alex Gramfort`_.
```

Comment:
>
> Seeing as this does actually improve this file a bit by giving it a better
> spacing, okay with you @agramfort <https://github.com/agramfort>
> @dengemann <https://github.com/dengemann> to merge it in this case? I
> think losing git blame on the whats_new is worth the improved spacing.
>
yeah ok. But next time let's do such cosmits outside of a feature branch.

besides the absence of plot tests looks ok to me. For the plot test just
make sure it runs. We cannot really check that it appears correctly on
screen.

thanks @teonlamont